config/application.rb

```diff
@@ -1,37 +1,19 @@
-require_relative "boot"
+require_relative 'boot'
 
-require "rails/all"
+require 'rails/all'
 
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
 module Transition
   class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration should go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded.
-
-    config.eager_load_paths += %W(#{config.root}/lib)
-
-    # Custom directories with classes and modules you want to be autoloadable.
-    config.autoload_paths += %W(#{config.root}/lib)
-
-    # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
-    # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    config.time_zone = "Europe/London"
 
-    config.active_record.belongs_to_required_by_default = false
-
-    # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
-    # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
-    # config.i18n.default_locale = :de
-
-    # TODO: this is no longer an encouraged pattern for dynamic error pages
```

Comment:
This TODO is a comment from 2016 that we've not looked at since. I don't know if we need these.

Reply:
We do need these. There's a failing Cucumber feature:

```
  Scenario: Visit the page of an non-existent site # features/site.feature:89
    Given I have logged in as a GDS Editor         # features/step_definitions/authentication_steps.rb:1
    When I visit the path /sites/not_a_site        # features/step_definitions/visit_steps.rb:5
    Then I should see our custom 404 page          # features/step_definitions/page_assertion_steps.rb:79
      Unable to find xpath "/html" (Capybara::ElementNotFound)
      ./features/step_definitions/page_assertion_steps.rb:2:in `/^I should see "([^"]*)"$/'
      ./features/step_definitions/page_assertion_steps.rb:80:in `/^I should see our custom 404 page$/'
      features/site.feature:92:in `Then I should see our custom 404 page'
```